helm-app-operator/hack/ci/setup-openshift.sh

```diff
@@ -0,0 +1,15 @@
+# Configure insecure docker registry for openshift
+sudo service docker stop
+sudo sed -i 's/DOCKER_OPTS=\"/DOCKER_OPTS=\"--insecure-registry 172.30.0.0\/16 /' /etc/default/docker
+sudo service docker start
+# Download oc to spin up openshift on local docker instance
+curl -Lo oc.tar.gz https://github.com/openshift/origin/releases/download/v3.10.0/openshift-origin-client-tools-v3.10.0-dd10d17-linux-64bit.tar.gz
```

Comment:
We can use `v3.11.0`
https://github.com/openshift/origin/releases/tag/v3.11.0

I think that's what we're testing the SDK against.